hello-world/hello-world.yaml

```diff
@@ -0,0 +1,70 @@
+# This describes what is deployed by this template.
+description: NFS server and client deployed with Heat on Chameleon
```

Comment:
Maybe we should update the description and remove the NFS bits from hello-world appliance, since we have a separate proper NFS appliance.